src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -436,6 +434,52 @@ void OrbitMainWindow::SetupTargetLabel() {
                    });
 }
 
+void OrbitMainWindow::SetupStatusBarLogButton() {
+  // The Qt Designer doesn't seem to support adding children to a StatusBar.
+  auto* capture_log_widget = new QWidget(statusBar());  // NOLINT
```

Comment:
Out of curiosity, what is the linter complaining about?

Reply:
clang-tidy complains (rightly so) that the result of `new` is not passed to an owner.

Reply:
Ah ok, thanks.